src/server/services/status.js

```diff
@@ -11,13 +11,25 @@ export async function getAllStatus() {
 }
 
 /**
+ * Fetch a service by its name.
+ *
+ * @param  {String} id
+ * @return {Promise}
+ */
+export async function getStatusGroupedByName() {
+  return await new Status().query(qb => qb.groupBy('name')
```

Comment:
database specific queries or logic should not be in the services. These should be in the Model class instead.
BTW, I think writing a SQL query would be more readable instead of using query builder. 
Can we create separate files just for storing SQL query as string constants like `queries/status.js`? 
These files would contain SQLs like these:

```javascript
export const FETCH_STATUS_QUERY = `
   SELECT s.*
   FROM service_names s
   GROUP BY s.name
   ORDER BY created_at DESC
`;
```

And the Model files would contain logic something like this:

```javascript
import { FETCH_STATUS_QUERY } from './queries/status';

class StatusChange extends db.Model {
  ...
  status fetchLatestStatuses() {
    return db.knex.raw(FETCH_STATUS_QUERY);
  }
  ...
}
```

And finally the services would contain this:

```javascript
export function fetchLatestStatuses() {
  return StatusChange.fetchLatestStatuses();
}
```
@leapfrogtechnology/chill-team